webapp/src/main/java/io/github/microcks/service/TraceSubscriptionService.java

```diff
@@ -0,0 +1,103 @@
+package io.github.microcks.service;
+
+import io.github.microcks.event.TraceEvent;
+import io.github.microcks.util.tracing.SpanFilterUtil;
+
+import io.opentelemetry.sdk.trace.ReadableSpan;
+import io.opentelemetry.sdk.trace.data.SpanData;
+import org.springframework.context.event.EventListener;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Service;
+import org.springframework.web.servlet.mvc.method.annotation.SseEmitter;
```

Comment:
I'm not a big fan of having the `Service` component being dependent on the web layer.... I think we should let this related logic into the `web` package.